templates/default.tpl

```diff
@@ -19,7 +19,7 @@ server {
 
     location ~ \.php$ {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
-        fastcgi_pass unix:/var/run/php5-fpm.sock;
+        fastcgi_pass unix:{% if php.ppa == 'php-7.0' %}/var/run/php/php7.0-fpm.sock{% else %}/var/run/php5-fpm.sock{% endif %};
```

Comment:
@popstas as I mentioned in the other PR, would it be possible to to use a dynamically set fact also in this case? I don''t like the fact that nginx has to access variables from a different role which might not exists.